guardrails/llm_providers.py

```diff
@@ -965,6 +970,14 @@ async def invoke_llm(self, *args, **kwargs) -> LLMResponse:
 def get_async_llm_ask(
     llm_api: Callable[[Any], Awaitable[Any]], *args, **kwargs
 ) -> AsyncPromptCallableBase:
+    try:
+        import litellm
+
+        if llm_api == litellm.acompletion or (llm_api is None and kwargs.get("model")):
+            return AsyncLiteLLMCallable(*args, **kwargs)
+    except ImportError:
+        pass
+
```

Comment:
this had to come up here because get_static_openai_acreate_funcr returns None and if  llm_api = None then we fall into the OpenAI code. i dont know how this will ever work if you do pass a callable though. since you couldnt even pass None llm_api before at the risk of maybe turning over some more rocks i dont want to know whats under them i think this maybe wrong but its all going away in 0.6.X anyways